bin/generate-npm-tag.sh

```diff
@@ -7,14 +7,18 @@ HIGHEST_PUBLISHED_VERSION=$(git tag --list 2>/dev/null | sort -V | tail -n1 2>/d
 # Extract tag version from ./packages/govuk-frontend/package.json
 CURRENT_VERSION=$(npm run version --silent --workspace govuk-frontend)
 
-function version() { echo "$@" | awk -F. '{ printf("%d%03d%03d\n", $1,$2,$3); }'; }
+version() { echo "$@" | awk -F. '{ printf("%d%03d%03d\n", $1,$2,$3); }'; }
 
-if [ $CURRENT_VERSION == $HIGHEST_PUBLISHED_VERSION ]; then
-  echo "⚠️ Git tag $TAG already exists. Check you have run `npm version` correctly."
+if [ "$CURRENT_VERSION" = "$HIGHEST_PUBLISHED_VERSION" ]; then
```

Comment:
Should that be a double equals? I'm worried I'm missing something as I'd expect this to be a declaration instead of a comparisson, but maybe this is doing something else?

Reply:
I'm not an expert on shell/bash, but essentially the [double equals version is Bash-specific](https://unix.stackexchange.com/questions/72039/whats-the-difference-between-single-and-double-equal-signs-in-shell-compari), whereas the single equals is "POSIX compliant", meaning it'll work in any generic shell script.

The fact that this script was running before means our GitHub runner has Bash on the go anyway, but thought I'd generify while I was in there.